src/main/build/atomic.h

```diff
@@ -137,7 +137,7 @@ static inline uint8_t __basepriSetRetVal(uint8_t prio)
 // On gcc 5 and higher, this protects only memory passed as parameter (any type can be used)
 // this macro can be used only ONCE PER LINE, but multiple uses per block are fine
 
-#if (__GNUC__ > 7)
+#if (__GNUC__ > 8)
```

Comment:
Have we actually verified that ATOMIC_BARRIER works as intended? @ledvinap would be the expert to ask.

Reply:
I did test it with gcc8 a bit some time ago, and did not see any problem.
Maybe the test can be removed completely, it is unlikely that semantics will change that much now and arduino world will be in much bigger trouble ...